src/test/java/teammates/test/cases/ui/StudentFeedbackSubmissionEditSaveActionTest.java

```diff
@@ -846,7 +872,8 @@ public void testGracePeriodExecuteAndPostProcess() throws Exception {
 
         String[] submissionParams = new String[]{
                 Const.ParamsNames.COURSE_ID, fs.getCourseId(),
-                Const.ParamsNames.FEEDBACK_SESSION_NAME, fs.getFeedbackSessionName()
+                Const.ParamsNames.FEEDBACK_SESSION_NAME, fs.getFeedbackSessionName(),
+                Const.ParamsNames.USER_ID, studentInGracePeriod.googleId
```

Comment:
Don't forget to remove all the user ID params on *all* places that you added them.